app/vmctl/backoff/backoff.go

```diff
@@ -11,8 +11,8 @@ import (
 )
 
 const (
-	backoffRetries     = 5
-	backoffFactor      = 1.7
+	backoffRetries     = 10
+	backoffFactor      = 1.6
 	backoffMinDuration = time.Second
```

Comment:
I have improved tests and made them configurable. These changes made tests faster. 
Also, I changed the consts for the backoff policy. With new values, it will have following delays
```
1 step 2s
2 step 3.8s
3 step 7.22s
4 step 13.718s
5 step 26.064s
6 step 49.521s
7 step 1m34s
8 step 2m58s
9 step 5m39s
10 step 10m45s 
```
If we are talking about the native migration process, the backoff policy is applied for export and import actions.
I mean, if we have errors when tries create an export pipe or import pipe backoff policy will try to repeat create these pipes.
In other migration processes, we have `Importer`, which tries to import batches to the storage. If this import failed backoff policy retries to import batch again

Reply:
Okay, I think datasources we requested data from are behaving the same as I described above. Thanks for update!